examples/hello_world/proto/BUILD

```diff
@@ -33,10 +35,22 @@ cc_proto_library(
     deps = [":hello_world_proto"],
 )
 
+java_proto_library(
```

Comment:
This should have a corresponding java_lite_proto_library that can be referenced by the java_grpc_library.